scripts/ebcli_installer.py

```diff
@@ -139,15 +139,17 @@ def _exec_cmd(args):
 
 
 INSTALLATION_SUCCESS_MESSAGE_WITH_EXPORT_RECOMMENDATION__NON_WINDOWS = """
-    Note: To complete installation, ensure `eb` is in PATH. You can ensure this by executing:
+    Note: To complete installation, ensure `eb` and 'python' are in PATH. You can ensure this by executing:
 
     1. Bash:
 
        echo 'export PATH="{eb_location}:$PATH"' >> ~/.bash_profile && source ~/.bash_profile
+       echo 'export PATH="{python_installation}:$PATH"' >> ~/.bash_profile && source ~/.bash_profile
```

Comment:
This is meaningless when you execute ebcli_installer.py and not bundled installer.py. you have to emit this message from bundled_installer.py and ensure the formatting matches

Reply:
Also, are you planning on eliminating the export message that the python_installer is going to emit?

Reply:
So this was a proposed solution to customers missing this statement when they were running the bundled_installer. For example, someone would start the installation, work on something else, come back to their window and only notice the final notice that says they needed to add eb to path. But you are right that someone who _only_ used the ebcli_installer may be unclear about the python statement. My thought on this would be to put a statement under this one we have called out that says something like 'If you have not already, add python to your path by doing the following...' Do you think that would be clear enough?

To your other point I don't think we should eliminate the notice from the python installer incase someone _only_ uses that script.

Reply:
You should avoid repeating the message. There are more elegant solutions to conditionally display the Python PATH export message. At any rate, the place for that is not the EBCLI installer.

Reply:
I see what you mean. I think at the risk of clogging up the docs changes I'm going to remove this change and submit a separate PR for a proposed solution to the above problem.